packages/lib-classifier/src/store/Classification/Classification.spec.js

```diff
@@ -132,31 +133,50 @@ describe('Model > Classification', function () {
         taskKey: 'T3',
         type: 'transcription'
       })
+      drawingTwo = drawingTask.TaskModel.create({
+        instruction: 'draw another thing',
+        taskKey: 'T4',
+        type: 'drawing'
+      })
     })
 
     it('should return an empty array if the stored annotations are not for drawing or transcription tasks', function ()  {
       model.addAnnotation(singleChoice, 0)
       model.addAnnotation(text, 'This is a text task')
-      expect(model.interactionTaskAnnotations).to.be.an('array')
-      expect(model.interactionTaskAnnotations).to.be.empty()
+      const previousAnnotations = model.previousInteractionTaskAnnotations('T1')
+      expect(previousAnnotations).to.be.an('array')
+      expect(previousAnnotations).to.be.empty()
       model.removeAnnotation('T0')
       model.removeAnnotation('T1')
     })
 
-    it('should return an array of annotations if the stored annotations are for drawing tasks', function () {
+    it('should return an empty array if the active task key is the same as the annotations task key', function () {
```

Comment:
This isn't quite true. It should return an empty array if all of our annotations are for the current task. I'm not quite sure how to word that.

Reply:
That's what I think I have worded here, but I can try to reword it.